packages/@n8n/nodes-langchain/nodes/document_loaders/DocumentRecursiveUrlLoader/DocumentRecursiveUrlLoader.node.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable n8n-nodes-base/node-dirname-against-convention */
+import {
+	NodeConnectionType,
+	type IExecuteFunctions,
+	type INodeType,
+	type INodeTypeDescription,
+	type SupplyData,
+} from 'n8n-workflow';
+import { RecursiveUrlLoader } from 'langchain/document_loaders/web/recursive_url';
+import type { CharacterTextSplitter } from 'langchain/text_splitter';
+import { compile } from 'html-to-text';
+import { logWrapper } from '../../../utils/logWrapper';
+import { getConnectionHintNoticeField } from '../../../utils/sharedFields';
```

Comment:
Could you please extract this node from this PR? We're having an ongoing discussion about document loaders so this one will have to wait a bit before getting merged.